Files.Launcher/Program.cs

```diff
@@ -170,7 +170,7 @@ private static void EndReadCallBack(IAsyncResult result)
             }
         }
 
-        private static async void Connection_RequestReceived(PipeStream conn, Dictionary<string, object> message)
+        private static async Task Connection_RequestReceived(PipeStream conn, Dictionary<string, object> message)
```

Comment:
`async void`?

Reply:
This method I have to call with Task.Run() so needs to return a Task.

Reply:
Ah right, could you break down this function into `RequestReceivedFromConnection()` and call that async Task function in Connectio_RequestReceived()